reports/imports/base.py

```diff
@@ -102,3 +102,20 @@ def slurp_year_grouped_table(self, ws, all_data, col_start, cols_per_group, cols
                     for irow in range(row_start, row_end + 1):
                         row_heading = canon(ws.cell(column=row_heading_col, row=irow).value)
                         col_data[row_heading] = v(ws.cell(column=iyear, row=irow).value)
+
+    def import_sheet_1(self, ws, data, col_start, col_end, row_end):
+        self.slurp_table(ws, data, col_start=col_start, col_end=col_end, row_end=row_end)
+
+    def import_sheet_5(self, ws, data, col_start, col_heading_row, row_start, row_end, row_heading_col):
+        self.slurp_year_grouped_table(
```

Comment:
Both of these functions are just one lines @ascii-dev , why not just call e.g. `slurp_table` directly? I don't think it's saving us anything defining them like this.